_posts/2018-07-26-discourse-6-months-on-impact-assesment.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: "Discourse, 6 months on: Impact Assessment"
+author:
+  name: "Greg Sutcliffe"
+  uri: gwmngilfen
+date: "2018-07-26"
+output:
+tags:
+- foreman
+- discourse
+- data-science
+---
+
+It's been 6 months since the Foreman community switched to Discourse, so I
+wanted to revisit some of the graphs I [drew in November][forum_post] (when I
```

Comment:
double [] and space before (), the link does not render

Reply:
Thanks @ares fixed